pkg/cloudprovider/fake/capacity.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/Pallinder/go-randomdata"
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why do we need to name the package when its already `v1`?

Reply:
I have no idea. I spent 10 minutes on this and couldn't figure out why. Does this not happen to you?